lib/lockdown/lockdown.js

```diff
@@ -0,0 +1,82 @@
+const LOCKDOWN_PORT = 62078;
+const LABEL = 'usbmuxd';
+const PROTOCOL_VERSION = 2;
+
+class Lockdown {
+
+  constructor (plistService) {
+    this.plistService = plistService;
+  }
+
+  async queryType () {
+    let data = await this.plistService.sendPlistAndReceive({
+      Label: LABEL,
+      ProtocolVersion: PROTOCOL_VERSION,
+      Request: 'QueryType'
+    });
+    if (data.Request === 'QueryType' && data.Type === 'com.apple.mobile.lockdown') {
+      return data;
+    } else {
```

Comment:
else is redundant

Reply:
Should I remove that from every place that I do this if-else check?